support-frontend/assets/components/orderSummary/__snapshots__/orderSummaryTsAndCs.test.tsx.snap

```diff
@@ -8,9 +8,9 @@ exports[`orderSummaryTs&Cs Snapshot comparison orderSummaryTs&Cs render product
 
 exports[`orderSummaryTs&Cs Snapshot comparison orderSummaryTs&Cs render product HomeDelivery for region GBPCountries for ratePlan Sunday correctly 1`] = `"Auto renews every month until you cancel. Cancel anytime.You will receive your first newspaper delivery on Sunday, September 7, 2025 as part of your Observer subscription."`;
 
-exports[`orderSummaryTs&Cs Snapshot comparison orderSummaryTs&Cs render product SubscriptionCard for region GBPCountries for ratePlan Sunday correctly 1`] = `"Auto renews every month until you cancel.Cancel anytime."`;
+exports[`orderSummaryTs&Cs Snapshot comparison orderSummaryTs&Cs render product SubscriptionCard for region GBPCountries for ratePlan Sunday correctly 1`] = `"Auto renews every month until you cancel. Cancel anytime.Your physical subscription card will be delivered to your door by Sunday, September 14, 2025, for you to collect in store the first newspaper edition you are entitled to in your Observer subscription."`;
```

Comment:
Is this extra copy about the physical subscription card an expected addition in this PR? I'm not sure how it relates.

Reply:
This is the old checkout copy (which now does not exist)  versus the new checkout copy again. See old here ->
<img width="690" height="405" alt="image" src="https://github.com/user-attachments/assets/6c4c0b2b-d1bf-4ef5-a37e-be071e604f7f" />